### DISCUSSION TITLE
Can't login with email account docker install

### DISCUSSION THREAD

**Discussion by tetra-archos** [the-djmaze/snappymail#1518] [Q&A]
I can verify my IMAP and SMTP settings with my login information, however when I try to login to the main page after doing so and saving it just spits a invalid login or password error. I'm unsure of how to proceed at this point since I can test and get a valid response but can't actually login. docker-compose through portainer is below as well as my SMTP and IMAP settings. Runbox is an email hosting service and I am using a custom domain with them. I have no trouble with any other SMTP and IMAP clients.

```
version: '2'
services:
  snappymail:
    image: djmaze/snappymail:latest
    ports:
      - 8888:8888
    environment:
      - DEBUG=true
    volumes:
      - snappymail:/opt/snappymail
    networks:
      - default
    restart: unless-stopped

  # This provides prometheus metrics for snappymail's php-fpm
  php-fpm-exporter:
    image: hipages/php-fpm_exporter:2.2.0
    ports:
      - 9253:9253
    environment:
      - PHP_FPM_SCRAPE_URI=tcp://snappymail:9000/status
    networks:
      - default

networks:
  default:

volumes:
  snappymail:
    driver: local
```

![image](https://github.com/the-djmaze/snappymail/assets/4935078/cdd22b1e-b588-48c7-aa9c-d7659018037a)

![image](https://github.com/the-djmaze/snappymail/assets/4935078/3babf638-c73e-4e60-b004-ce763d6715ce)

**Comment by cm-schl** [comment]
Hi,
we would need some logs to see whats going on: https://github.com/the-djmaze/snappymail/wiki/FAQ#how-do-i-enable-logging

You also are saying that you use a `custom domain` so maybe have a look if you need to define an alias for your domain: https://github.com/the-djmaze/snappymail/wiki/Admin-Manual#add-alias

**Reply by tetra-archos** [reply]
I figured it out, I'm just an idiot and forgot I enabled app passwords.
